types/happo-cypress/index.d.ts

```diff
@@ -0,0 +1,34 @@
+// Type definitions for happo-cypress 1.17
+// Project: https://github.com/happo/happo-cypress#readme
+// Definitions by: Alexander Polyankin <https://github.com/me>
```

Comment:
<!--
  1 failure:  This line should ...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/happo-cypress/index.d.ts;
  Line: 3;
-->
- :no_entry_sign: This line should have your github username in it, not /me